unit_test/live_model_test.cc

```diff
@@ -57,6 +57,7 @@ namespace {
   const auto JSON_CONTEXT = R"({"_multi":[{},{}]})";
   const auto JSON_CONTEXT_WITH_SLOTS = R"({"_multi":[{},{}],"_slots":[{}]})";
   const auto JSON_CONTEXT_WITH_SLOTS_WITH_SLOT_IDS = R"({"_multi":[{},{}],"_slots":[{"_id":"provided_slot_id_1"}, {}]})";
+  const auto JSON_CONTEXT_WITH_SLOTS_W_SLOT_IDS_AND_SLOT_NS = R"({"_multi":[{},{}],"_slots":[{"_id":"provided_slot_id_1", "slot_namespace":{"a":"b"}}, {}]})";
```

Comment:
[nit] maybe we can also add third slot with {"slot_namespace":{"_id":"ignored_slot_id"}}?

Reply:
sure